Codyception: Coding Cody Tools With Cody For Coding With Cody

```diff
@@ -0,0 +1,142 @@
+
+
+## Examples of Time-Saving Commands ##
+
+![](http://cody.kynlo.co.uk/2.png)
+
+
+    "implement-code-localization-support": {
+      "description": "implement-code-localization-support",
+      "prompt": "Implement code localization support in the current project, enabling adaptation to different languages and cultural preferences.",
+      "context": {
+        "codebase": true
+      }
+    },
+    "check-code-code-smells": {
+      "description": "check-code-code-smells",
+      "prompt": "Check and identify code smells in the current project, suggesting refactoring for improved code quality.",
+      "context": {
+        "codebase": true
+      }
+    },
+    "evaluate-code-external-apis": {
+      "description": "evaluate-code-external-apis",
+      "prompt": "Evaluate and report on the usage and reliability of external APIs in the current project, suggesting updates or replacements.",
+      "context": {
+        "codebase": true
+      }
+    },
+    "implement-code-database-indexing": {
+      "description": "implement-code-database-indexing",
+      "prompt": "Implement code database indexing strategies in the current project, optimizing data retrieval and query performance.",
+      "context": {
+        "codebase": true
+      }
+
+## The cody.json File: Tailoring Commands to Your Needs ##
+
+Customizing commands is made easy with the `cody.json` file. Here's how you can leverage it:
+
+### Project-Specific Commands:
+
+Create `cody.json` in the `.vscode` directory of your project to make commands accessible only in that workspace. Cody will execute these custom commands when working on the project.
+
+### Global Commands:
+
+Create a new `cody.json` file in the `.vscode` directory of your home directory for commands accessible across multiple projects. Cody will execute these global custom commands in any workspace.
+
+### Workspace vs. User Commands
+
+Understanding the scope of your custom commands is essential:
+
+- `cody.json` in a project's `.vscode` folder: Commands are available only in that workspace.
+- `cody.json` in the home `.vscode` folder: Commands are available globally across all workspaces.
+
+This flexibility allows you to organize project-specific commands separately from reusable commands accessible globally, enhancing your customization capabilities with Cody.
+
+
+
+
+## Cody's Command Syntax ##
+
+### Running Commands Made Simple ###
+Executing commands is a breeze with these options:
+
+- Type / in the chat box and select your desired command from the pop-up list.
+
+- Use ctrl+shift+v (cmd+shift+v on apple) anywhere in your document to open the Command Menu, and just like magic, you can select and run commands without the chat sidebar.
```

Comment:
```suggestion
- Use ctrl+shift+v (cmd+shift+v on mac OS) anywhere in your document to open the Command Menu, and just like magic, you can select and run commands without the chat sidebar.
```

Reply:
I will make the requested changes and "hone" it in.  I can set to work on this in the morning, so if there's any specific content you want adding/deleting, or something you want specifically pointed out, feel free to let me know and I'll add it to my notes, otherwise, expect the changes by the end of the day (GMT)! :)